tools/auto_labeling_3d/scripts/id_list_example.txt

```diff
@@ -0,0 +1,2 @@
+772cb8c8-a289-4e70-bae2-fc088b6ad279
```

Comment:
@jacoblambert 

Thank you so much for your PR!!
What do you think about using yaml instead of txt?